### DISCUSSION TITLE
Live updating property of child component into parent component

### DISCUSSION THREAD

**Discussion by tushargugnani** [livewire/livewire#6114] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://gist.github.com/tushargugnani/ffd194a53f065bccd29e2f7041d42805

### Additional information

I have a parent component with property `foo` and a child component with property `bar` , I have `wire:model.live`  in the child property  and have `Modelable` attribute on the child component's property. 

However, changing the child components property does not change the parent components property live. It get;s changed after some other update happens on the component.

Is this how it is supposed to be? Any way I can make the modelable property live?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@tushargugnani thanks for reporting! Any chance you could submit a failing test PR so we can investigate? Thanks!

**Comment by tushargugnani** [comment]
@joshhanley Thank you, I tried creating a test PR, but I am not able to get past  the error `failed to connect to localhost port 9515 after 1 ms: Connection refused`, I tried resolving this by running `./vendor/bin/dusk-updater update ` but not able to resolve it. Any sort of help / tips are appreciated.

**Reply by nuernbergerA** [reply]
what OS?

**Reply by tushargugnani** [reply]
@nuernbergerA OSX

**Reply by nuernbergerA** [reply]
ok what i have to do on my laptop is to copy `phpunit.xml.dist` to `phpunit.xml`
and add the `DISPLAY` env

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit colors="true"
         stopOnFailure="false"
         backupGlobals="false"
         processIsolation="false"
         backupStaticAttributes="false"
         bootstrap="vendor/autoload.php"
         convertDeprecationsToExceptions="true"
         convertErrorsToExceptions="true"
         convertNoticesToExceptions="true"
         convertWarningsToExceptions="true">
    <testsuites>
        <testsuite name="Unit">
            <directory suffix="UnitTest.php">./src</directory>
        </testsuite>
        <testsuite name="Browser">
            <directory suffix="BrowserTest.php">./src</directory>
        </testsuite>
        <testsuite name="LegacyBrowser">
            <directory suffix="Test.php">./legacy_tests/Browser</directory>
        </testsuite>
    </testsuites>
    <php>
        <env name="DISPLAY" value=":1" force="true"/>
    </php>
</phpunit>
```

**Reply by joshhanley** [reply]
@tushargugnani Sometimes I find you just need to close any open chrome windows and then it runs.

**Comment by tushargugnani** [comment]
@joshhanley @nuernbergerA I reinstalled dusk and now when I run a dusk test, and now instead of `connection refused` I am getting `operation timed out` on port 9515. I am using MAC M1 (if that matters). Doesn't look like a generic issue, I will keep looking for solutions.

**Comment by charlesobrien** [comment]
@joshhanley @tushargugnani @nuernbergerA Can confirm I experienced the same issue as well — child component with wire:model.live and a modelable property does not update parent component property unless some other action occurs (e.g., an event).

Sorry I don’t have a failing test case to submit yet either; but just wanted to leave the note that I experienced the issue/could reproduce.
